spec/models/manageiq/providers/hawkular/inventory/parser/middleware_servers_spec.rb

```diff
@@ -98,35 +101,37 @@ def parsed_server
       :feed      => 'feed1',
       :hostname  => 'standalone',
       :product   => 'WildFly Full',
-      :type_path => 'WildFly Server'
+      :type_path => 'WildFly Server WF10'
     )
     expect(parsed_server.properties).to include(eap_properties_hash)
     expect(parsed_server.properties).to include('In Container' => 'true')
     expect(parsed_server.properties).to_not include('Immutable')
   end
 
   it 'assigns status reported by inventory to a server with "up" metric' do
-    metric_data.data.first['value'] = 'up'
+    metric_data['value'][1] = '1'
 
     parser.parse
-    expect(parsed_server.properties['Availability']).to eq('up')
+    expect(parsed_server.properties['Availability']).to eq('Running')
     expect(parsed_server.properties['Calculated Server State']).to eq(parsed_server.properties['Server State'])
   end
 
-  it 'assigns status reported by metric to a server when its availability metric is something else than "up"' do
+  it 'assigns STOPPED to a server when its availability metric is something else than "up"' do
+    metric_data['value'][1] = '0'
+
     parser.parse
-    expect(parsed_server.properties['Availability']).to eq('arbitrary value')
-    expect(parsed_server.properties['Calculated Server State']).to eq('arbitrary value')
+    expect(parsed_server.properties['Availability']).to eq('STOPPED')
+    expect(parsed_server.properties['Calculated Server State']).to eq('STOPPED')
   end
 
-  it 'assigns unknown status to a server with a missing metric' do
+  it 'assigns STOPPED to a server with a missing metric' do
```

Comment:
@lucasponce @jshaughn I'm wondering why this behavior changed.
IIRC, this was the case when managing a remote server. But I'm not sure, since I never set up that case. So, aren't we supporting that case now?

Reply:
Point is the avail metric has changed.
So, when there is a missing metric, that means it can't reach the server, so it can be read as stopped.
I think know unknown status is used when there is not an availability result.

Reply:
Right, the avail metric has changed but perhaps here @israel-hdez is correct.  This is the test that completely omits an avail metric (i.e. so, no no data points).  Having said that, I'm not sure it matters too much as STOPPED and Unknown will both indicate that something is wrong.  So, I'd suggest that we approach this in a top-down way, and if we get a complaint from QE, or perhaps a user, we can revisit the logic.

Reply:
It's OK for me to go in a top-down way. I also see it as not too important. I mainly was looking for feedback for the new behavior since, now, I feel again like a starter due the new metrics backend :)